polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -550,6 +552,140 @@ impl pallet_treasury::Config for Runtime {
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
 }
 
+// Council
+pub type CouncilInstance = pallet_collective::Instance1;
```

Comment:
Can you please be more detailed about the council here? Is it the technical committee?

Reply:
Haha, I have seen the committee now, but still, the uninvolved will never know what the council is.

Reply:
From the PR description:
> No council added. A technical committee is included.
Is the description outdated?

Reply:
Hm. It's the name Kusama uses:
![grafik](https://user-images.githubusercontent.com/73821294/167559512-006f4aaa-48d9-4a08-9bfc-dd61f2accb22.png)

What name would you like for the Council instead? Acala uses GeneralCouncil within the code.


Ah, yes, it's outdated. Sorry, let me adapt.

Reply:
More info why we finally ended with council: https://github.com/integritee-network/parachain/issues/108

Reply:
I think the name is ok. But to a new dev, it might be good to have in 1-2 sentences, what a council should do? Maybe I am too pedantic here... You decide if this should be added.

Reply:
Sure, I can add some comment